sklearn/tests/test_multioutput.py

```diff
@@ -405,7 +405,7 @@ def test_multi_output_exceptions():
     with pytest.raises(NotFittedError):
         moc.predict(y)
 
-    with pytest.raises(NotFittedError):
+    with pytest.raises(AttributeError):
```

Comment:
Now, in addition, we should as well check that the same error is raised after calling `fit`